### ISSUE TITLE
"Ephemeral" spec options

### ISSUE THREAD

**Issue by hazmat345** [loganasherjones/yapconf#57]
* Yapcon version: 0.2.3
* Python version: 3.6
* Operating System: Centos 7 

I think it'd be very helpful to have a flag that would prevent an item from being written to configuration file.

I have a configuration file option in my spec to make it easy for users to specify one from the command line. But this doesn't really make sense _inside_ the file.

**Comment by loganasherjones** [comment]
Hmm, I think I would like to use the `bootstrap` identifier for this. I could see a flag on `migrate_config_file` that allows you to specify `include_bootstrap`

**Comment by hazmat345** [comment]
I agree, a bootstrap-type item is the only place where I think this makes sense.

You may want the flag on `generate_config` (is that the name?) as well to prevent bootstrap entries from showing up there.
